darglint/config.py

```diff
@@ -260,6 +266,7 @@ def load_config_file(filename):  # type: (str) -> Configuration
         strictness=strictness,
         ignore_regex=ignore_regex,
         ignore_raise=ignore_raise,
+        ignore_properties=ignore_properties,
```

Comment:
If the user doesn't specify `ignore_properties` in their config file, this will raise an error for a variable referenced before assignment.  Add a default value at the beginning of this function.